.gitlab-ci.yml

```
@@ -300,6 +300,7 @@ run-certification-tests:
           echo "Deploying to $DEVICE_UNDER_TEST"
           tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/prplmesh.ipk
       fi
+    - ci/git-clean-reset.sh /easymesh_cert "$(<"ci/easymesh_cert_version")"
```

regarding my previous comment on adding a comment to `easymesh_cert_version` - it will require some "hulahoop" here, something along those lines (please verify) -
```suggestion
    - ci/git-clean-reset.sh /easymesh_cert "$(<"grep -m1 ^[^#] ci/easymesh_cert_version")"
```

---

Which is exactly why adding a redundant comment is not a good idea.

---

Having only the commit hash in this file makes it easy to use, so I prefer to keep it like that.
I agree we should add any comments that could be useful (like why the commit hash has to be updated) to the commit message.